examples/react-native-expo-example/metro.config.js

```diff
@@ -12,7 +12,11 @@ const config = getDefaultConfig(projectRoot);
 const ALIASES = {
 	"@noble/hashes/crypto": path.resolve(
 		monorepoRoot,
-		"node_modules/@noble/hashes/crypto.js"
+		"node_modules/@noble/hashes/crypto.js",
+	),
+	"@sinclair/typebox": path.resolve(
```

Comment:
Interesting that we aren't seeing the issue here with `ox/erc6492/WrappedSignature.ts` since this metro config doesn't have my fix from the other example.  Do you know if we updated the `ox` package.  I wonder if there was something specific that fixed the ox package.

Curious to get your thoughts, I think this might be a fix where we won't have to hardcode each individual package.  Happy to move forward with your fix though if you think the Alias approach is better.

https://github.com/alchemyplatform/react-native-expo-example/blob/da3f3ff392ae46655b23da786de212b3fd8752ce/metro.config.js#L36-L58

Reply:
Yeah good questions - I was just thinking we've seen if for two packages so far.  But we probably aren't going to be adding a lot more internal packages going forward so it might not be an issue